quiz.html

```diff
@@ -1,3 +1,4 @@
+<!-- Nice job, Thadius. I have a few comments below. -->
 <!doctype html>
```

Comment:
First, make sure your image is named properly (no spaces) and make sure the image tag has a / before the closing >